app/src/main/res/values-cn/strings.xml

```diff
@@ -37,4 +37,9 @@
     <string name="example_2_meaning">从早上6点到6点15分的每一分钟</string>
     <string name="example_3_meaning">每个月最后一天的上午8时15分</string>
     <string name="example_4_meaning">2022年每个月的第三个星期五，从早上7点开始到早上7点55分结束，每5分钟一次</string>
+    
```

Comment:
This change is not consistent with the other language files. I would also prefer the omission of the blank lines.

Reply:
I removed the lines though technically speaking, localization work should and only should follow raw strings content, including number of lines.

Reply:
I lack the experience to assess whether the blank lines are a convention. If this is so, then they are also missing in the other language files. This is what I meant by lack of consistency.